torch/utils/data/datapipes/iter/callable.py

```diff
@@ -131,6 +136,33 @@ def __setstate__(self, state):
             self.fn = dill_function  # type: ignore[assignment]
 
 
+def _collate_helper(conversion, item):
+    # TODO(VitalyFedyunin): Verify that item is any sort of batch
```

Comment:
Do we need to check the type of `item` here?

Reply:
And, do we want to extend to support list of dictionary?

Reply:
As TODO line